package.json

```diff
@@ -27,7 +27,7 @@
     "style-loader": "^1.2.1"
   },
   "scripts": {
-    "start": "webpack-dev-server --open --hot --colors",
```

Comment:
- Colors does not do anything
- Hot interfered with webpack config which already had hot reloading
- open is already part of webpack config